### PR TITLE
[Edit] C++: Syntax Style Fix

### PR DESCRIPTION
**Pull request by yangc95**
### Description

Editing code blocks to adhere to Codecademy style: https://www.codecademy.com/courses/c-plus-plus-for-programmers/articles/cpp-style-guide

Also in compliance with Google's style guide: https://google.github.io/styleguide/cppguide.html#cpplint

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@yangc95, Looks good to me.

File: content/cpp/concepts/loops/loops.md [hunk 1]
```diff
@@ -22,8 +22,7 @@ A `while` loop statement repeatedly executes the code block within as long as th
 Note that the while loop might not ever run. If the condition is false initially, the code block will be skipped.
```
Comment (Dusch4593) [inline content/cpp/concepts/loops/loops.md:22]:
```suggestion
> **Note:** The `while` loop might not ever run. If the condition is initially `false`, the code block will be skipped.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@yangc95 Thank you for making these edits in the C++ topic! 🚀

I think it will be good to achieve more consistency in syntax between our Docs content and the code snippets used in our Catalog. 😄

Should we also consider empty lines above and below lines inside `if/else` statements (as shown in [this lesson](https://www.codecademy.com/courses/learn-c-plus-plus/lessons/cpp-conditionals-and-logic/exercises/else))? Other than the `conditionals.md` file, there are other files that feature `if/else` statements.

**Review by KTom101 — COMMENTED** [review]
@yangc95 I added a couple minor changes.

**Comment by yangc95** [comment]
@Dusch4593 @KTom101 hope the changes applied are satisfactory! I'll try going over these one more time prior to merge.

Thanks for parsing through all these C++ entries 💟

**Review by Dusch4593 — APPROVED** [review]

**Comment by yangc95** [comment]
@Dusch4593 thanks for merging!

**Pull request merged**